Add support for artillery processor configuration

gcuni opened the pull request:
Artillery has the concept of [processor](https://artillery.io/docs/testing_http.html#advanced-writing-custom-logic-in-javascript) for adding custom logic before/after query executions. I noticed this doesn't work in serverless-artillery. Digging a bit further it seems that artillery converts the processor value in the script into functions outside of the artillery-core project. I'm not sure why they do that, but the relevant few lines are of code can be found [here](https://github.com/shoreditch-ops/artillery/blob/master/lib/worker.js#L66). In order to get this feature to work in serverless-artillery, the equivalent procedure has to happen in the handler.js lambda. So I extended it to do just that and now I can have custom Javascript in my tests which is great!
One small issue is that the pre-commit script does not allow dynamic require so current code fails pre-commit. If you want to support this feature you will have to relax that.

[edit: direct link to the line of code]

erikerikson commented on lib/lambda/handler.js:
```diff
@@ -727,6 +728,17 @@ const impl = {
+    const config = script.config;
+    if (config.processor) {
+      const processorPath = path.resolve(process.cwd(), config.processor);
+      config.processor = require(processorPath);
```
add `// eslint-disable-line <rule>[, <rule>[, ...]]` as you feel appropriate.  This is a perfect place to require the custom module so disabling that rule for this line seems appropriate.

gcuni commented:
I added some tests for this. While adding them, I realized that because of the way the processor loading works it would be a problem if for some reason loadProcessor were called multiple times on same script. So I added an additional guard against that and a test to cover it.
Also, saw your comment on that artillery-core project. Hopefully this will be addressed in core in the future so you can always have feature parity with artillery.

erikerikson commented:
Thanks @gcuni, appreciate the tests!  Great changes.
